uitools/import/Esri/ArcGISRuntime/Toolkit/Controller/SearchViewController.qml

```diff
@@ -14,6 +14,509 @@
+                        if (internal.results.count > 0) {
+                            for (let j = internal.results.count - 1; j >= 0; j--) {
+                                const result = internal.results.get(j);
+                                if (result && result.owningSource === source) {
+                                    internal.results.remove(j);
+                                }
+                            }
+                        }
+                        // Go through all suggestions and remove those that come from this source.
+                        if (internal.suggestions.count > 0) {
+                            for (let k = internal.suggestions.count - 1; k >= 0; k--) {
+                                const suggestion = internal.suggestions.get(k);
+                                if (suggestion && suggestion.owningSource === source) {
+                                    internal.suggestions.remove(k);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        property ListModel suggestions: ListModel {
+            function element(index) {
+                return this.get(index.row);
+            }
+        }
+
+        property ListModel results: ListModel {
+            function element(index) {
+                return this.get(index.row);
+            }
+        }
+
+        property Connections geoViewConnections: Connections {
+            target: controller.geoView
+            function onViewpointChanged() {
+                if (!controller.automaticConfigurationEnabled) {
+                    return;
+                }
+
+                // update queryArea/queryCenter with the viewpoint change.
+                if (geoView instanceof SceneView) {
+                    const vp = geoView.currentViewpointCenter;
+                    if (vp) {
+                        internal.queryCenter = vp.center;
+                        internal.queryArea = null;
+                    }
+                } else if (geoView instanceof MapView) {
+                    const vp2 = geoView.currentViewpointExtent;
+                    if (vp2) {
+                        internal.queryCenter = vp2.extent.center;
+                        internal.queryArea = vp2.extent;
+                    }
+                }
+
+                // Show the requery button if the user manually navigated the view.
+                if (geoView.navigating) {
+                    controller.eligableForRequery = true;
+                }
+            }
+            function onNavigatingChanged() {
+                console.log("Navigating", geoView.navigating);
+            }
+        }
+
+        property Component defaultWorldGeocoderSource: Component {
+            LocatorSearchSource {
+                locatorTask: LocatorTask {
+                    url: 'https://geocode-api.arcgis.com/arcgis/rest/services/World/GeocodeServer'
```

Comment:
I recommend sticking with either single or double quotes for strings for consistency. There are a mix in this PR.

Reply:
Good point. The JS style guide suggests [single quotes](https://google.github.io/styleguide/jsguide.html#features-string-literals) so I'll update with that if that makes sense.

Reply:
Single quotes it is, sounds good.